lib/src/components/NxIconDropdown/__tests__/NxIconDropdown.test.tsx

```diff
@@ -40,38 +41,38 @@ describe('NxIconDropdown', () => {
   });
 
   it('renders a button with the appropriate classes and type=button', function() {
-    const component = getShallowComponent(),
+    const component = getMountedComponent(),
         button = component.find(NxButton),
         defaultIcon = component.find(NxFontAwesomeIcon);
 
-    expect(component).toHaveClassName('.nx-icon-dropdown');
+    expect(component.find(AbstractDropdown)).toHaveClassName('.nx-icon-dropdown');
     expect(defaultIcon).toHaveProp('icon', faEllipsisV);
     expect(button).toHaveClassName('.nx-icon-dropdown__toggle');
     expect(button).toHaveProp('variant', 'icon-only');
     expect(button).toHaveProp('type', 'button');
   });
 
   it('correctly renders a custom icon based on icon prop', function() {
-    const component = getShallowComponent({ icon: faCog });
+    const component = getMountedComponent({ icon: faCog });
     expect(component.find(NxFontAwesomeIcon)).toHaveProp('icon', faCog);
   });
 
   it('correctly renders the menu based on isOpen prop', function() {
-    let component = getShallowComponent({ isOpen: true });
+    let component = getMountedComponent({ isOpen: true });
     expect(component.find(NxDropdownMenu)).toExist();
 
-    component = getShallowComponent({ isOpen: false });
+    component = getMountedComponent({ isOpen: false });
     expect(component.find(NxDropdownMenu)).not.toExist();
   });
 
   it('correctly assigns supplied classes', function() {
-    const component = getShallowComponent({ className: 'class1 class2' });
+    const component = getMountedComponent({ className: 'class1 class2' }).find(AbstractDropdown);
```

Comment:
This should go down to the div IMO

Reply:
> This should go down to the div IMO

Updated: 49aa3582e3f83104b1fe96ee9d123b417d169b53